docs/workflows/flatpak.rst

```diff
@@ -0,0 +1,62 @@
+.. _flatpak-workflow:
+
+Hosting Flatpak Content in OCI Format
+=====================================
+
+Pulp can host Flatpak application and runtime images that are distributed in OCI format.  To make
+such content discoverable, it can provide ``/index/dynamic`` and ``/index/static`` endpoints as
+specified by `the Flatpak index protocol
+<https://github.com/owtaylor/flagstate/blob/master/docs/protocol.md>`_.  This is not enabled by
+default.  To enable it, define ``FLATPAK_INDEX = True`` in the settings file.
+
+Clients like the ``flatpak`` command-line tool or the GNOME Software application will typically
+query the ``/index/static`` endpoint, which is intended to be called repeatedly with identical query
+parameters, and whose responses are meant to be cached.  The ``/index/dynamic`` endpoint serves
+exactly the same content, but is intended for one-off requests that should not be cached.  These
+endpoints can be accessed without authentication.  They only provide information about public
+distributions.
+
+The two endpoints support a number of query parameters (``architecture``, ``tag``, ``label``, etc.),
+see the protocol specification for details.  Two notes:
+
+* Every request must include a ``label:org.flatpak.ref:exists=1`` query parameter.  This acts as a
+  marker to only report Flatpak content, and to exclude other container content that may also be
+  provided by the Pulp instance.
+
+* This implementation does not support annotations, so including any ``annotation`` query parameter
+  will result in an empty response.  Use ``label`` query parameters instead.
```

Comment:
result in 400

Reply:
can you add a note that this is in line for flatpack clients works?

Reply:
done, <https://github.com/pulp/pulp_container/pull/1301/commits/1f6b2cbe768b0c3e7871a3956385b8ab36d0d3b9> "Make annotation query parameters cause 400 response" and <https://github.com/pulp/pulp_container/pull/1301/commits/73b728389570ba329b51a46afbad0e55498eec89> "Documentation improvements"